ASPNetCoreWebAPITraining/Controllers/PersonController.cs

```diff
@@ -44,7 +45,7 @@ public async Task<ActionResult<Person>> GetPerson(int id)
         public async Task<IActionResult> PutPerson(int id, Person person)
         {
             using var context = _dbContextFactory.CreateDbContext();
-            if (id != person.Id)
+            if (id != ((BaseEntity)person).Id)
```

Comment:
@christopherjomelztan I thought there was a casting issue because its not showing up but it turns out that the is issue with my intellisense. I have reverted this line of code to its initial value.